package.json

```diff
@@ -17,10 +17,7 @@
     "xkp": "cli.js",
     "xkpasswd": "cli.js"
   },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/vot/xkpasswd-node.git"
-  },
+  "repository": "https://github.com/vot/xkpasswd-node.git",
```

Comment:
Yeah that does make sense. So only corner case is skipping `test` directory from publishing? I agree its not a lot of extra files, so should ok to keep it.